src/position.cc

```diff
@@ -595,3 +595,16 @@ void Position::print() const
         std::cout << std::endl;
     }
 }
+
+bool Position::insufficient_material() const
+{
+	if(bb::count(occ()) > 3)
+		return false;
+	if(bb::count(occ(KNIGHT)) == 1)
+		return true;
+	if(bb::count(occ(BISHOP)) == 1)
+		return true;
+	if(bb::count(occ()) == 2)
+		return true;
+	return false;
+}
```

Comment:
OK, I cut down a few lines. It's functional integrity was retained.
I could only get it shorter if I added a function to the Position class like Minors() to return the bishop and knight occupancy.
I think it's fine the way it is though.